### DISCUSSION TITLE
Issues with flet CLI

### DISCUSSION THREAD

**Discussion by IgorDzilla** [flet-dev/flet#2604] [Q&A]
### Question

When I try to use any of flet CLI tools on Windows 10,
 I get "bash: flet: command not found" error, but it perfectly works on MacOS.
![Screenshot 2024-02-11 121801](https://github.com/flet-dev/flet/assets/150725317/f7014b9c-b50e-43f9-9864-2d12e480f542)

However, the script itself works.

### Code sample

flet pack -h

### Error message

bash: flet: command not found

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by 09u2h4n** [comment]
Are other flet commands work?
Like `flet -h`, if it doesn't work maybe you use flet module in the virtual environment.?

**Reply by IgorDzilla** [reply]
None of them work. Trying to install flet via pip results in "Requirement already satisfied".

**Reply by 09u2h4n** [reply]
Let's try in virtual environment.

1) Create virtual environment
`python3 -m venv flet_env`

2) Activate it
`flet_env/Scripts/activate`

3) Install flet
`pip3 install flet`

4) Try the command
`flet -h`

5) For deactivating the environment
`deactivate`

If 4th command still doesn't work then the pip doesn't installs flet or fletd, check it (e.g., C:\Python310\Scripts\flet.exe -h) to find where flet is type:
`pip3 show flet`

```
Name: flet
Version: 0.19.0
Summary: Flet for Python - easily build interactive multi-platform apps in Python
Home-page:
Author: Appveyor Systems Inc.
Author-email: hello@flet.dev
License: Apache-2.0
Location: /usr/lib/python3.11/site-packages
Requires: cookiecutter, flet-runtime, packaging, qrcode, watchdog, websocket-client, websockets
Required-by:
```
You will see location then type
`python3 <flet location>/flet/cli/cli.py -h`
Replace the location that you found in this case it will be
`python3 /usr/lib/python3.11/site-packages/flet/cli/cli.py -h`
The output
```
usage: cli.py [-h] [--version]
              {create,run,pack,publish,build} ...

positional arguments:
  {create,run,pack,publish,build}
    create              Create a new Flet app from a
                        template.
    run                 Run Flet app.
    pack                Package Flet app to a standalone
                        bundle.
    publish             Publish Flet app as a standalone
                        web app.
    build               Build an executable app or
                        install bundle.

options:
  -h, --help            show this help message and exit
  --version             show program's version number and
                        exit
```
